tools/rna_tools/inforna/.shed.yml

```diff
@@ -0,0 +1,9 @@
+name: inforna
+owner: rnateam
+description: INFO-RNA is a server for the design of RNA sequences that fold into a given pseudo-knot free RNA secondary structure.
```

Comment:
INFO-RNA is a service ...

Like in the tool's <help>. Sounds better because it is not a standalone server